random_menu.rb

```diff
@@ -0,0 +1,34 @@
+puts "How many items would you like to enter?"
+chosen_num = gets.chomp.to_i
+adj = []
+style = []
+food = []
+
+chosen_num.times do
+  puts "Please enter an adjective"
+  adj << gets.chomp
+end
+chosen_num.times do
+  puts "Please enter a style"
+  style << gets.chomp
+end
+chosen_num.times do
+  puts "Please enter a food"
+  food << gets.chomp
+end
+
+adj2 = adj.clone
+style2 = style.clone
+food2 = food.clone
+
+puts "How many menu items (#{chosen_num} or under) would you like to display?"
+number = gets.chomp.to_i
+until number<=chosen_num
+  puts "Choose a number smaller than #{chosen_num}!"
+  number = gets.chomp.to_i
+end
+x=1
+number.times do
+  puts "#{x} #{adj2.delete_at(rand(adj2.length))} #{style2.delete_at(rand(style2.length))} #{food2.delete_at(rand(food2.length))}"
+  x+=1
+end
```

Comment:
I like that you use the value returned by `delete_at` here - very clever. I would comment that this line is long and difficult to read, and it might be worthwhile to break it up into several statements, something like:

```ruby
adjective = adj2.deelete_at(rand(adj2.length))
style = style2.delete_at(rand(style2.length))
food = food2.delete_at(rand(food2.length))
puts "#{x} #{adjective} #{style} #{food}"
```